common/src/main/java/com/wynntils/models/worlds/BombModel.java

```diff
@@ -114,11 +128,17 @@ public BombInfo getLastBomb() {
     private static final class ActiveBombContainer {
         private final Map<BombKey, BombInfo> bombs = new ConcurrentHashMap<>();
 
-        public void add(BombInfo bombInfo) {
+        public void add(BombInfo bombInfo, boolean replaceIfExists) {
             BombKey key = new BombKey(bombInfo.server(), bombInfo.bomb());
 
             // Ensure no duplicate bombs are added
-            if (bombs.containsKey(key) && bombs.get(key).isActive()) return;
+            if (bombs.containsKey(key)) {
+                if (replaceIfExists) {
+                    bombs.remove(key);
+                } else if (bombs.get(key).isActive()) {
+                    return;
+                }
+            }
```

Comment:
```suggestion
            removeOldTimers();
            
            if (bombs.containsKey(key) && !replaceIfExists) {
                return;
            }
```

And `bombs.put` will override the old bomb.